modules/cert-manager-acme-dns-providers.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-operator-issuer-acme.adoc
+
+:_content-type: CONCEPT
+[id="cert-manager-acme-dns-providers_{context}"]
+= Supported DNS-01 providers
+
+The {cert-manager-operator} supports the following DNS-01 providers for ACME issuers:
+
+* Amazon Route 53
+* Azure DNS
+* Google Cloud DNS
+
+// TODO: Do I need to put both known issue references here too?
```

Comment:
@bergerhoffer not sure if you need put those bugs both here and in security/cert_manager_operator/cert-manager-operator-release-notes.adoc , I'm fine to put them in both places. You can confirm with other stakeholders may be, e.g. Anjali.

Reply:
You can put in release docs and put a link here pointing to it?

Reply:
Unfortunately we have a limitation where we can't link to other areas of our docs from our modules. I think I might just put a quick summary of it here too to raise awareness of these limitations